<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NysseHajos - Settings</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>168</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>465</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Game mode</string>
   </property>
   <widget class="QRadioButton" name="timeGoalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>442</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Time goal: collect as many points as possible in 5 minutes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="passengerGoalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>455</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger goal: collect 40 passengers as quickly as possible</string>
    </property>
   </widget>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>118</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="timeCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Use custom time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
